manifest.json

```diff
@@ -1,27 +1,28 @@
 {
-
-  "manifest_version": 2,
-  "name": "wxIF",
-  "version": "0.2",
+  "applications": {
```

Comment:
Why is this needed? What's the point of it?

We don't use anything requiring fixed IDs. I don't see why we care what the installed filename is.

Reply:
Answered below in my comment below for `manifest.json`. It's mainly for easier finding, but also good practice, too.